helm/cluster-cloud-director/values.schema.json

```diff
@@ -559,6 +435,129 @@
                         }
                     }
                 },
+                "controlPlane": {
```

Comment:
We might want to add it to required section like https://github.com/giantswarm/cluster-cloud-director/pull/312/files#diff-e2aa539a2b05339dcc4afe7112e00ef491c55342eef3f5e79d21ba5d12e8be7aL124